src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -658,9 +704,17 @@ private void HighlightBookButton(BookInfo bookInfo)
 			foreach (var btn in AllBookButtons())
 			{
 				if (btn.Tag == bookInfo)
+				{
 					btn.FlatAppearance.BorderColor = Palette.TextAgainstDarkBackground;
+					_bookTriangle.Left = btn.Left + btn.Width - _bookTriangle.Width - 2;
```

Comment:
Is there a better name for this method now it is doing more than highlighting the indicated button? DisplayBookButtonAsActive?